### ISSUE TITLE
Regarding the issue of training metrics for AIGCDetectBenchmark Progan

### ISSUE THREAD

**Issue by lovegd-asyou** [shilinyan99/AIDE#27]
I used the same training script and made no other modifications. base_dir="/seu_nvme/home/220252364/judgement/AIDE/train_class"
torchrun \
--nproc_per_node 1 \
--nnodes 1 \
--master_port 29505 \
/seu_nvme/home/220252364/judgement/AIDE/main_finetune.py \
--resnet_path "/seu_nvme/home/220252364/judgement/AIDE_our/resnet50-19c8e357.pth" \
--convnext_path "/seu_nvme/home/220252364/judgement/AIDE_our/open_clip_pytorch_model.bin" \
--batch_size 130 \
--blr 1e-4 \
--epochs 20 \
--output_dir ${base_dir} \
--log_dir ${base_dir} \
--data_path "/seu_share/home/220252364/dataset/AIDE_CLASS/progan_train" \
--eval_data_path "/seu_share/home/220252364/dataset/AIDE_CLASS/test" \
2> ${base_dir}/error.log 1>${base_dir}/logs.log
After several epochs, the mean accuracy began to decline and failed to reach the performance described in the paper. Could you please tell me where the error might be?

{"train_lr": 5.07786304123261e-05, "train_loss": 0.23767979066172626, "train_class_acc": 0.9891468965043055, "train_min_lr": 5.07786304123261e-05, "test_loss": 0.5558900009078456, "test_acc1": 75.90385891815451, "test_acc5": 100.0, "epoch": 0, "n_parameters": 54432466}
{"train_lr": 5.076291203383308e-05, "train_loss": 0.21532221244741248, "train_class_acc": 0.9963892483607938, "train_min_lr": 5.076291203383308e-05, "test_loss": 0.3683272960285346, "test_acc1": 86.2042039547151, "test_acc5": 100.0, "epoch": 1, "n_parameters": 54432466}
{"train_lr": 5.073148378575147e-05, "train_loss": 0.21160123621401622, "train_class_acc": 0.9970669595100622, "train_min_lr": 5.073148378575147e-05, "test_loss": 0.35336238008435117, "test_acc1": 86.86634627273635, "test_acc5": 100.0, "epoch": 2, "n_parameters": 54432466}
{"train_lr": 4.709174377446268e-05, "train_loss": 0.20996936735510288, "train_class_acc": 0.997344709966432, "train_min_lr": 4.709174377446268e-05, "test_loss": 0.38198807171973165, "test_acc1": 85.49190874479763, "test_acc5": 100.0, "epoch": 3, "n_parameters": 54432466}

the result in the paper：

<img width="1566" height="511" alt="Image" src="https://github.com/user-attachments/assets/fc0d3805-5156-4609-a16f-9b91cbad2013" />

**Comment by kmy6156** [comment]
I encountered a similar problem to yours. Have you solved it?
